src/main/java/frc/trigon/robot/subsystems/pitcher/Pitcher.java

```diff
@@ -57,6 +61,22 @@ public Command getPitchingCommand(DoubleSupplier targetAngle) {
                 .andThen(this::stop, this);
     }
 
+    /**
+     * @param targetAngle the command will continuously set the target angle to this, if this is 0, then the default
+     *                    target angle will be used.
+     * @return a command that sets the angle of the pitcher according to the given supplier.
+     */
+    public Command getPitchingCommandWithDefault(DoubleSupplier targetAngle) {
```

Comment:
rename to `getPitchingCommandWithDefaultAngle`

Reply:
Maybe default value?

Reply:
no coz you also have default velocity

Reply:
Change to IdleAngle